types/nano/index.d.ts

```diff
@@ -854,6 +854,9 @@ declare namespace nano {
     // Return the documents in descending by key order. Default is false.
     descending?: boolean;
 
+    // Stop returning records when the specified key is reached.
```

Comment:
Please update the comment to clarify that this is an alias for `end_key` in order to avoid confusion.